scripts/globals/weaponskills.lua

```diff
@@ -959,3 +959,61 @@ function takeWeaponskillDamage(defender, attacker, params, finaldmg, slot, tpHit
 
     return finaldmg;
 end
+
+function applyAftermathEffect(player, tp, spower_1, spower_2, spower_3)
+    spower_1 = spower_1 or 1
+    spower_2 = spower_2 or 1
+    spower_3 = spower_3 or 1
+
+    -- AFTERMATH LEVEL 1
+
+    if ((tp >= 1000) and (tp <= 1100)) then
```

Comment:
So, I started to fix that, but then I noticed that the power param of addStatusEffect is not linear.  For level 1, it goes from 10-19, incrementing by 1, then jumps to 24 for the first 100 tp of level 2, and increases by 4 for evey level until the last one, where it only increases by 3.  Then, the level 3 aftermath effect then goes backward and applies a power level of 45, instead of something greater than 59.  Instead of trying to guess what the intention was, I decided to leave it alone.  If someone can explain what's going on there, I'll do my best to fix it.